apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/AutomationRuleEvaluatorsResource.java

```diff
@@ -0,0 +1,168 @@
+
+        return Response.ok().entity(evaluator).build();
+    }
+
+    @POST
+    @Operation(operationId = "createAutomationRuleEvaluator", summary = "Create automation rule evaluator", description = "Create automation rule evaluator", responses = {
+            @ApiResponse(responseCode = "201", description = "Created", headers = {
+                    @Header(name = "Location", required = true, example = "${basePath}/api/v1/private/automation/evaluators/project/{projectId}/evaluator/{evaluatorId}", schema = @Schema(implementation = String.class))
+            })
+    })
+    @RateLimited
+    public Response createEvaluator(
+            @RequestBody(content = @Content(schema = @Schema(implementation = AutomationRuleEvaluator.class))) @JsonView(AutomationRuleEvaluator.View.Write.class) @NotNull @Valid AutomationRuleEvaluator evaluator,
+            @Context UriInfo uriInfo) {
+
+        String workspaceId = requestContext.get().getWorkspaceId();
+        String userName = requestContext.get().getUserName();
+
+        log.info("Creating {} evaluator for project_id '{}' on workspace_id '{}'", evaluator.type(),
+                evaluator.projectId(), workspaceId);
+        AutomationRuleEvaluator savedEvaluator = service.save(evaluator, workspaceId, userName);
+        log.info("Created {} evaluator '{}' for project_id '{}' on workspace_id '{}'", evaluator.type(),
+                savedEvaluator.id(), evaluator.projectId(), workspaceId);
+
+        URI uri = uriInfo.getAbsolutePathBuilder()
+                .path("/projectId/%s/evaluator/%s".formatted(savedEvaluator.projectId().toString(),
+                        savedEvaluator.id().toString()))
+                .build();
+        return Response.created(uri).build();
+    }
+
+    @PUT
+    @Path("/evaluator/{id}")
+    @Operation(operationId = "updateAutomationRuleEvaluator", summary = "update Automation Rule Evaluator by id", description = "update Automation Rule Evaluator by id", responses = {
+            @ApiResponse(responseCode = "204", description = "No content"),
+    })
+    @RateLimited
+    public Response updateEvaluator(@PathParam("id") UUID id,
+            @PathParam("projectId") UUID projectId,
+            @RequestBody(content = @Content(schema = @Schema(implementation = AutomationRuleEvaluatorUpdate.class))) @NotNull @Valid AutomationRuleEvaluatorUpdate evaluatorUpdate) {
+
+        String workspaceId = requestContext.get().getWorkspaceId();
+        String userName = requestContext.get().getUserName();
+
+        log.info("Updating automation rule evaluator by id '{}' and project_id '{}' on workspace_id '{}'", id,
+                projectId, workspaceId);
+        service.update(id, projectId, workspaceId, userName, evaluatorUpdate);
+        log.info("Updated automation rule evaluator by id '{}' and project_id '{}' on workspace_id '{}'", id, projectId,
+                workspaceId);
+
+        return Response.noContent().build();
+    }
+
+    @DELETE
```

Comment:
We're generally creating batch endpoints, so you can delete by sending multiple ids.